demo/src/index.js

```diff
@@ -56,7 +92,7 @@ class Demo extends Component {
       <div>
         <ReactSearchKit
           searchConfig={{
-            url: 'https://zenodo.org/api/records/',
+            url: 'https://zenodo.org/api/records/',//,'https://localhost:5000/api/documents/'
```

Comment:
@vladbmihaescu can you please remove this comment?

Reply:
Done!